tests/Integration/HtmlPurifierTest.php

```diff
@@ -23,7 +22,7 @@ public function setUp(): void {
 
 	public function testReturnsAllAllowedTags(): void {
 		$this->assertSame(
-			'<h1>my <u>test</u> <em>site</em></h1>
+			'<h1>my <u>tests</u> <em>site</em></h1>
```

Comment:
These are test strings and not path references, so I'd prefer to keep them singular

Reply:
yeah, we use the "test" folder usually not only for having particular "tests" in it, but also test related setups, configs, other test environment related stuff, which makes "test" a better abstract parent category and it's a widely used standard